src/NServiceBus.RabbitMQ/Routing/DefaultRoutingKeyConvention.cs

```diff
@@ -68,9 +36,28 @@ static string GetRoutingKey(Type type, string key = "")
             return key + type.FullName.Replace(".", "-");
         }
 
-        static readonly byte[] MsPublicKeyToken = typeof(string).Assembly.GetName().GetPublicKeyToken();
+        static bool IsSystemType(Type type)
+        {
+            bool result;
+
+            if (!IsSystemTypeCache.TryGetValue(type, out result))
+            {
+                var nameOfContainingAssembly = type.Assembly.GetName().GetPublicKeyToken();
+                IsSystemTypeCache[type] = result = IsClrType(nameOfContainingAssembly);
+            }
+
+            return result;
+        }
+
+        static bool IsClrType(byte[] a1)
+        {
+            IStructuralEquatable structuralEquatable = a1;
+            return structuralEquatable.Equals(MsPublicKeyToken, StructuralComparisons.StructuralEqualityComparer);
+        }
+
+        static bool IsNServiceBusMarkerInterface(Type type) => type == typeof(IMessage) || type == typeof(ICommand) || type == typeof(IEvent);
```

Comment:
With indentation, the line is question is 143 characters long, and doesn't even reach half way on my monitor in Visual Studio. :smile: 

I've never been a huge fan of wrapping stuff to multiple lines just because, especially if it's a single expression.

If we were going to enforce some sort of line length I'd personally prefer it to be in relation to how the code looks in VS than how it shows up on GitHub.